Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SB-2-A.yml

```diff
@@ -24,7 +24,7 @@ benefit:
 
 #A one sentence description of the activity
 shortDescription:
-  Automate your build pipeline, secure the used tooling. Add security checks in the build pipeline.
+  Automate your build pipeline and secure the used tooling. Consider adding security checks in the build pipeline.
```

Comment:
I'm not sure if I would weaken it. The idea was: 
- Level 2: You have some checks in the pipeline so that the results are available, but the quality is not enforced.
- Level 3: You enforce that the results are reviewed and break the build if defined criteria is not met.

Reply:
I agree. Changed the phrasing again.